lib/flagsmith-es/src/flagsmith-core.ts

```diff
@@ -80,6 +82,18 @@ const Flagsmith = class {
 
         return _fetch(url, options)
             .then(res => {
+                const lastUpdated = res.headers?.get('x-flagsmith-document-updated-at');
```

Comment:
Is there a way to avoid having to define this code multiple times? Seems like it's already defined here:

https://github.com/Flagsmith/flagsmith-js-client/pull/163/files#diff-0b9b511548a3e7571b5371b6ad98facf6955df9dfa855af7107b182e04bc524bR85

Reply:
Ah, the files in lib are generated sourcemaps. I think eventually I'll gitignore these, but will need to come up with a way to build them when doing local development.

Reply:
lib/flagsmith-es is essentially what the user gets in node_modules when they npm install flagsmith-es

Reply:
gotcha, ok